airflow/cli/commands/variable_command.py

```diff
@@ -66,44 +68,82 @@ def variables_delete(args):
     Variable.delete(args.key)
 
 
+DIS_RESTRICT = 'restrict'
+DIS_OVERWRITE = 'overwrite'
+DIS_IGNORE = 'ignore'
+CREATED = 'created'
+DISPOSITIONS = [DIS_RESTRICT, DIS_OVERWRITE, DIS_IGNORE]
+
+
+def _prep_import_status_msgs(var_status_map):
```

Comment:
Can you add some assertions that will let you see what the output of this function is?

Reply:
what's the intent of these assertions?
to check for valid status messages, ex -  assert len(msg) > 1?